### ISSUE TITLE
Error starting file: mssql-jdbc-9.2.1.jre8.jar

### ISSUE THREAD

**Issue by peterjohnlawrence** [ontop/ontop#433]
### Description

Installed Ontop, and then downloaded MSSQL JDBC jar. However I get this message when starting up Ontop

### Steps to Reproduce

1. Download Ontop for windows
2. Download MSSQL JDBC jar
3. I  preferences add MSSQL JDBC
4. Restart

This error is then reported on startup:

```
Error starting file:/D:/Program%20Files/Ontop/Protege-5.5.0/plugins/mssql-jdbc-9.2.1.jre8.jar
org.osgi.framework.BundleException: Unable to resolve com.microsoft.sqlserver.mssql-jdbc [21](R 21.0): missing requirement [com.microsoft.sqlserver.mssql-jdbc [21](R 21.0)] osgi.wiring.package; (&(osgi.wiring.package=org.osgi.service.jdbc)(version>=1.0.0)(!(version>=2.0.0))) Unresolved requirements: [[com.microsoft.sqlserver.mssql-jdbc [21](R 21.0)] osgi.wiring.package; (&(osgi.wiring.package=org.osgi.service.jdbc)(version>=1.0.0)(!(version>=2.0.0)))]
        at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4111) ~[org.apache.felix.main.jar:na]
        at org.apache.felix.framework.Felix.startBundle(Felix.java:2117) ~[org.apache.felix.main.jar:na]
        at org.apache.felix.framework.Felix.setActiveStartLevel(Felix.java:1371) ~[org.apache.felix.main.jar:na]
        at org.apache.felix.framework.FrameworkStartLevelImpl.run(FrameworkStartLevelImpl.java:308) [org.apache.felix.main.jar:na]
        at java.lang.Thread.run(Thread.java:745) [na:1.8.0_121]
```

----------------------------------- Platform -----------------------------------

Java: JVM 1.8.0_121-b13  Memory: 506M
Language: en, Country: US
Framework: Apache Software Foundation (1.8)
OS: windows10 (10.0)
Processor: x86

----------------------------------- Platform -----------------------------------

Protege Desktop
Version 5.5.0, Build

------------------------------------ Plugins -----------------------------------
Plugin: Ontop OBDA Prot├⌐g├⌐ plugin (4.1.0)
Plugin: Microsoft JDBC Driver for SQL Server (9.2.1) was not successfully started.  Please see the Prot├⌐g├⌐ log for more details.

**Comment by ghxiao** [comment]
Hi @peterjohnlawrence ,

Could you try to put the jdbc jar in a different directory? Protege treats all jar files under `plugins` as Protege plugins.

**Comment by peterjohnlawrence** [comment]
Yes, that was that error, thank you.

We also had problems using the MS JDBC driver connecting to a SQLServer. FYI we used the jTDS driver and eventually got Ontop to connect.

**Comment by kontchakov** [comment]
But what exactly was the problem with he MS JDBC driver?

**Comment by peterjohnlawrence** [comment]
We started with the MS JDBC driver but ere getting 18546(?) errors, even though we believed we had the correct JVM (32bit) and placed the correct ntlauth.dll in windows/System32.
We swapped to the jTDS driver and got a bit further, and eventually solved our connection issue when we put the ntlauth.dll in the protege/jre/bin folder.
However we did not test out doing the same for the MS jdbc driver (location of the ntauth.dll), so cannot confirm that it did *not* work. But since we now have it working with the jTDS driver, we did not want to risk messing with it:-)
